sys/include/net/netopt.h

```diff
@@ -574,6 +577,84 @@ typedef enum {
      */
     NETOPT_PHY_BUSY,
 
+    /**
```

Comment:
as @miri64 pointed out, I think this could be represented with NETOPT_ADDRESS_LONG. In IEEE802.15.4 is also an EUI, so we can reuse that one